scripts/CMakeLists.txt

```diff
@@ -10,15 +10,19 @@ ie_shellcheck_process(DIRECTORY "${OpenVINO_SOURCE_DIR}"
                       SKIP "${OpenVINO_SOURCE_DIR}/bin"
                            "${OpenVINO_SOURCE_DIR}/build"
                            "${OpenVINO_SOURCE_DIR}/thirdparty"
+                           "${OpenVINO_SOURCE_DIR}/src/plugins/intel_cpu/thirdparty"
```

Comment:
Why do we need changes in this file?

Reply:
I removed all changes except /src/plugins/intel_cpu/thirdparty, but the locally spellchecker is failed on
"${IE_MAIN_SOURCE_DIR}/temp"
"${OpenVINO_SOURCE_DIR}/scripts/install_dependencies/install_NCS_udev_rules.sh"
"${OpenVINO_SOURCE_DIR}/src/tests/functional/plugin/myriad/run_tests_myriad_multistick.sh".
I think, it happened due to my spellchecker version.

> shellcheck --version
ShellCheck - shell script analysis tool
version: 0.7.0